README.md

```diff
@@ -4,38 +4,37 @@ A site displaying "Hello Microverse"
 
 ![screenshot](images/Hello-Microverse.gif)
 
-It displays a blue hello world text, centred vertically and horizontally using margin and text-align properties in CSS 
+It displays a blue hello world text, centred vertically and horizontally using margin and text-align properties in CSS
 
 **Built With**
 
-●HTML
+● HTML
 
-●CSS
+● CSS
 
-●Linters
+● Linters
 
 **Live Demo**
 
 **Getting Started**
 
 To get a local copy up and running follow these simple example steps.
 
--clone the app by running this command 
+-clone the app by running this command
 
-:white_large_square: ```git clone  https://github.com/fobadara/Greetings.git```
+:white_large_square: `git clone https://github.com/fobadara/Greetings.git`
 
-:white_large_square: ```cd  greetings```
+:white_large_square: `cd greetings`
 
 :white_large_square: Open in browser
 
-
 **Prerequisites**
 -You will be needing:
 -A web browser to view output
 -A text editor
 
 **Run tests**
--You can run tests by reloading on the pull request I have created or simple create a pull rrequest of your own 
+-You can run tests by reloading on the pull request I have created or simple create a pull rrequest of your own
 **Authors**
 👤 Abdulfatai Badara
 
```

Comment:
- [ ] Let us arrange our text in our Readme file clear as provided in the Microverse README template.

------------------------------------Yours-----------------------------------------

![readme](https://user-images.githubusercontent.com/55635977/128846757-aca4c410-7931-49f0-8489-7ff834a1461f.png)

-----------------------------------Microverse Template---------------------------------------

![readme1](https://user-images.githubusercontent.com/55635977/128846766-d9c59836-40c8-497f-a0b2-dec5c4899e9b.png)